src/JsWebSocket.js

```diff
@@ -65,4 +65,10 @@ exports._wsReconnect = ws => () => {
   ws.reconnect();
 };
 
-exports._wsClose = ws => () => ws.close();
+exports._wsTerminate = ws => () => {
+  if (typeof BROWSER_RUNTIME == "undefined" || !BROWSER_RUNTIME) {
+    ws.close();
```

Comment:
Shouldn't this be the other way around? IIRC Node's `ws` has both `terminate` and `close` methods, but the browser only has `close`

Reply:
You are right. It's strange that the error does not appear in the console.